sdk/identity/identity/test/manual-integration/Cloudshell/index.ts

```diff
@@ -0,0 +1,21 @@
+import { SecretClient } from "@azure/keyvault-secrets";
+import { DefaultAzureCredential } from "@azure/identity";
+
+async function main(): Promise<void> {
+  // DefaultAzureCredential expects the following three environment variables:
+  // - AZURE_TENANT_ID: The tenant ID in Azure Active Directory
+  // - AZURE_CLIENT_ID: The application (client) ID registered in the AAD tenant
+  // - AZURE_CLIENT_SECRET: The client secret for the registered application
+  const credential = new DefaultAzureCredential();
+
+  const url = "https://" + process.env.KEY_VAULT_NAME + ".vault.azure.net";
+  const client = new SecretClient(url, credential);
+
+  await client.setSecret("secret-name130", "secret-value130");
```

Comment:
When last I tried, CloudShell MSI didn't support AADv2 scopes (which are used by our track 2 clients); does this work?

Reply:
This worked okay for me. Were you expecting an error?

Reply:
You're making me doubt myself. I'll run it again and double-check.

Reply:
You were right. I'd copied the wrong set of notes. I've updated them and confirmed that the test works.